src/ray/protobuf/gcs.proto

```diff
@@ -283,9 +283,9 @@ message JobConfig {
   uint32 num_java_workers_per_process = 2;
   // The jvm options for java workers of the job.
   repeated string jvm_options = 3;
-  // A list of directories or jar files that specify the search path for user
-  // code. This will be used as `CLASSPATH` in Java, and `PYTHONPATH` in
-  // Python.
+  // A list of directories or jar files or dynamic libraries that specify the search path
+  // for user code. This will be used as `CLASSPATH` in Java, and `PYTHONPATH` in Python.
+  // In C++, libraries in the path will be loaded by 'dlopen'.
```

Comment:
```suggestion
  // A list of directories or files (jar files or dynamic libraries) that specify the search path
  // for user code. This will be used as `CLASSPATH` in Java, and `PYTHONPATH` in Python.
  // In C++, libraries in the path will be loaded by 'dlopen'.
```